python/private/internal_config_repo.bzl

```diff
@@ -79,6 +85,7 @@ def _internal_config_repo_impl(rctx):
 
     rctx.file("rules_python_config.bzl", _CONFIG_TEMPLATE.format(
         enable_pystar = enable_pystar,
+        enable_deprecation_warnings = enable_deprecation_warnings,
```

Comment:
nit: Just inline the bool_from_environ() call here. The other one's aren't inlined because they have more complex logic to determine their value.